QuasarHelios2015/src/org/team1540/quasarhelios/DriveCode.java

```diff
@@ -16,8 +16,20 @@
 	private static FloatOutput rightBackMotor = Igneous.makeTalonMotor(3, Igneous.MOTOR_FORWARD, .1f);
 	private static FloatOutput rightMotors = FloatMixing.combine(rightFrontMotor, rightBackMotor);
 	private static FloatOutput leftMotors = FloatMixing.combine(leftFrontMotor, leftBackMotor);
+	public static FloatOutput allMotors = FloatMixing.combine(leftMotors, rightMotors);
+	public static FloatOutput rotate = new FloatOutput() {
+		public void set(float value) {
+			leftMotors.set(value);
+			rightMotors.set(-value);
+		}
+	};
 	public static BooleanStatus octocanumShifting = new BooleanStatus();
-	
+	public static FloatInput leftEncoderRaw = FloatMixing.createDispatch(Igneous.makeEncoder(6,7, Igneous.MOTOR_REVERSE), Igneous.globalPeriodic);
+	public static FloatInput rightEncoderRaw = FloatMixing.createDispatch(Igneous.makeEncoder(8,9, Igneous.MOTOR_FORWARD), Igneous.globalPeriodic);
```

Comment:
Using `MOTOR_FORWARD` and `MOTOR_REVERSE` here is a bit unexpected. This is fine if it's your intention. (Perhaps I should add `ENCODER_FORWARD` and `ENCODER_REVERSE` to the CCRE.)